Sample/Kitchen Sink/DemonstrationTools/CrossWindCalculator.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Shape+Extensions.swift
+//  flight-ui-ios
+//
+//  Created by Appivate 2023
+//
+
+import SwiftUI
+import Foundation
+import FlightUI
+
+struct CrossWindCalculator: View {
+
+    @StateObject var viewModel = CrosswindCalculatorViewModel()
+    @EnvironmentObject var theme: Theme
+
+    var body: some View {
+        ScrollView {
+            VStack {
+                HStack() {
+                    runwayNumber
```

Comment:
Minor/Opinion: As in previous PR, I'd probably group the entry fields into a single var, instead of wrapping each individual input field 👍